test/node/src/insert.test.ts

```diff
@@ -485,12 +527,72 @@ for (const dialect of BUILT_IN_DIALECTS) {
           sqlite: NOT_SUPPORTED,
         })
 
-        await query.execute()
+        const result = await query.execute()
+
+        expect(result).to.have.length(1)
+        expect(result[0]).to.containSubset({
+          species: 'hamster',
+          name: 'Catto',
+        })
       })
     }
 
+    it('should insert multiple rows', async () => {
+      const query = ctx.db.insertInto('person').values([
+        {
+          first_name: 'Foo',
+          last_name: 'Bar',
+          gender: 'other',
+        },
+        {
+          first_name: 'Baz',
+          last_name: 'Spam',
+          gender: 'other',
+        },
+      ])
+
+      testSql(query, dialect, {
+        postgres: {
+          sql: 'insert into "person" ("first_name", "last_name", "gender") values ($1, $2, $3), ($4, $5, $6)',
+          parameters: ['Foo', 'Bar', 'other', 'Baz', 'Spam', 'other'],
+        },
+        mysql: {
+          sql: 'insert into `person` (`first_name`, `last_name`, `gender`) values (?, ?, ?), (?, ?, ?)',
+          parameters: ['Foo', 'Bar', 'other', 'Baz', 'Spam', 'other'],
+        },
+        sqlite: {
+          sql: 'insert into "person" ("first_name", "last_name", "gender") values (?, ?, ?), (?, ?, ?)',
+          parameters: ['Foo', 'Bar', 'other', 'Baz', 'Spam', 'other'],
+        },
+      })
+
+      const result = await query.executeTakeFirst()
+
+      expect(result).to.be.instanceOf(InsertResult)
+      expect(result.numInsertedOrUpdatedRows).to.equal(2n)
+
+      if (dialect === 'postgres') {
+        expect(result.insertId).to.be.undefined
+      } else {
+        expect(result.insertId).to.be.a('bigint')
+      }
+
+      const inserted = await ctx.db
+        .selectFrom('person')
+        .selectAll()
+        .orderBy('id', 'desc')
+        .limit(2)
+        .execute()
+
+      expect(inserted).to.have.length(2)
```

Comment:
Surely the length is two since you use `limit(2)` 😄

Reply:
Whoops! 🤣